spec/functions/number.md

```diff
@@ -222,18 +223,18 @@ If the _operand_ of the _expression_ is an implementation-defined numeric type,
 such as the _resolved value_ of an _expression_ with a `:number` or `:integer` _annotation_,
 it can include option values.
 These are included in the resolved option values of the _expression_.
-The `:math` _options_ are not included in the resolved option values.
+The `:offset` _options_ are not included in the resolved option values.
```

Comment:
Why?

Reply:
At least two reasons:
1. Clarity; the function processes and "consumes" the option, as its resolved value includes the offset. This also allows for an implementation to present the resolved value of an `:offset` and a `:number` using the same representation.
2. We've not identified any use cases that would benefit from a consumer of an `:offset` value to know that an offset was applied to the value, or what that offset was.